crates/cxx-qt-gen/src/generator/structuring/qobject.rs

```diff
@@ -31,4 +33,25 @@ impl<'a> StructuredQObject<'a> {
             signals: vec![],
         }
     }
+
+    pub fn method_lookup(&self, id: &Ident) -> Result<Name> {
+        let method = self
+            .methods
+            .iter()
+            .find(|method| method.name.rust_unqualified() == id);
+
+        if let Some(method) = method {
+            Ok(method.name.clone())
+        } else {
+            Err(Error::new_spanned(
+                id,
+                format!("Method with name '{id}' not found!"),
+            ))
+        }
```

Comment:
This looks like it could be simplified to something like this?

```rust
method
  .map(|method| method.name.clone())
  .ok_or_else(|| Error:new_spanned(...));
```

Reply:
Done